test.Dockerfile

```diff
@@ -13,6 +13,12 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
  && apt-get install -y yarn
 
 RUN apt-get install -y firefox-esr
+RUN apt-get install -y postgresql-9.6
```

Comment:
We use 10.6 in AWS

Reply:
but 9.6 in Travis 🤷‍♂ .. I followed Travis, I'll see if I can bump it up...

Reply:
`E: Unable to locate package postgresql-10.6` 😞

Reply:
ah well. I doubt our tests are using anything specific to 10.6

Reply:
in the worst case scenario, staging would blow and the pipeline stops ;)